jcl/src/java.base/share/classes/jdk/internal/foreign/abi/UpcallLinker.java

```diff
@@ -42,9 +42,11 @@
  * The counterpart in OpenJDK is replaced with this class that wrap up
  * an upcall handler enabling the native call to the java code at runtime.
  */
+@SuppressWarnings("nls")
```

Comment:
Please don't do this; use `//$NON-NLS-n$` comments instead (like line 150).

Reply:
Removed.